test/e2e/config/docker.yaml

```diff
@@ -86,6 +109,7 @@ providers:
 variables:
   # default variables for the e2e test; those values could be overridden via env variables, thus
   # allowing the same e2e config file to be re-used in different prow jobs e.g. each one with a K8s version permutation
+  INIT_WITH_BINARY: "https://github.com/kubernetes-sigs/cluster-api/releases/download/v0.3.16/clusterctl-{OS}-{ARCH}"
```

Comment:
How would we run the test without the binary (I assume we want to be able to do that), can we overwrite it with an empty string via env vars?

Reply:
For this test, we need two versions of clusterctl.
One is embedded in the framework and is clusterctl v1alpha4. The other is a downloaded binary

Other tests will simply ignore this variable

Reply:
Okay I think I was mainly confused because of the "sometimes optional" nature of the variable in the test, but I think I got it now.